auslib/web/admin/swagger/api.yaml

```diff
@@ -168,49 +193,499 @@ paths:
+      tags:
+        - Rules
+      operationId: auslib.web.admin.views.mapper.rules_id_or_alias_delete
+      consumes: []
+      produces:
+        - application/json
+        - text/html
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#delete"
+        description: "Deletes the rule identified by the rule_id."
+      parameters:
+        - name: id_or_alias
+          in: path
+          description: rule_id or alias field value to update the rule against
+          type: string
+          required: true
+        - name: csrf_token
+          in: query
+          description: valid csrf token
+          type: string
+          required: true
+        - name: data_version
+          in: query
+          description: Name of the product to filter the rules against.
+          type: string
+          required: true
+
+      responses:
+        '200':
+          description: Successfully deleted a rule
+        '404':
+          description: Requested rule did not exist
+          schema:
+            $ref: '#/definitions/4xxClientErrorResponseModel'
+          examples:
+            application/json:
+              detail: Requested rule does not exist
+              status: 404
+        default:
+          description: Default case for sporadic exceptions like value error or permission denied error or requires remote username and password in basic authentication for posting new data.
+          examples:
+            application/json:
+              Error: "REMOTE_USERNAME required"
+
+  /rules/{rule_id}/revisions:
+    get:
+      summary: Fetches all revisions of a Rule
+      description: Fetches list and count of all the revisions of the rule given by rule_id
+      tags:
+        - Rules
+      operationId: auslib.web.admin.views.mapper.rules_revisions_get
+      consumes: []
+      produces:
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/database.html#rules"
+        description: "Fetches list and count of all the revisions of the rule given by rule_id"
+      parameters:
+        - name: rule_id
+          in: path
+          description:  Field value to uniquely identfiy the rule
+          type: integer
+          minimum: 0
+          required: true
+        - name: page
+          in: query
+          description: Variable value used pagination of response revisions.
+          type: integer
+          minimum: 1
+          x-nullable: true
+          required: false
+        - name: limit
+          in: query
+          description: Variable value used for calculating offset for pagination.
+          type: integer
+          minimum: 1
+          x-nullable: true
+          required: false
+
+      responses:
+        '200':
+          description: Returns JSON object of list of rules with their count
+          schema:
+            type: object
+            allOf:
```

Comment:
Just making sure I understand this correctly: RulesListGET and HistoryModel combine to form the full schema?